src/client/scripts/api/pageContext.ts

```diff
@@ -1,4 +1,4 @@
-import { IPageContext } from "../../../../@types/app/shared";
```

Comment:
this wasn't working for me locally ("module not found: TypeScript compiler error), so I changed it from being a .d.ts to a regular .ts file and import that instead.

Reply:
@marcj I think using `import type { IPageContext } from "../../../../@types/app/shared";` works here 👍